Src/VimCore/CoreInterfaces.fs

```diff
@@ -5228,6 +5228,12 @@ and IVimBuffer =
     [<CLIEvent>]
     abstract Closed: IDelegateEvent<System.EventHandler>
 
+    /// Raised after the buffer is closed AND any pending KeyInputEnd has been raised.
+    /// A buffer can be closed internally (i.e. while processing its own keyboard input), which will raise Closed -> KeyInputEnd -> InputClosed.
+    /// It can also be closed externally, which will raise Closed -> InputClosed.
+    [<CLIEvent>]
+    abstract InputClosed: IDelegateEvent<System.EventHandler>

```

Comment:
Instead of `InputClosed` I think we should call this `PostClosed`. This matches the naming convention `ITextBuffer` used for their events: `Changing`, `Changed` and `PostChanged`. That has symetry with the closing events here and the three variants are needed for the same reason.